doc/man3/OSSL_HTTP_transfer.pod

```diff
@@ -190,13 +190,20 @@ Since this function is typically called by applications such as
 L<openssl-s_client(1)> it uses the B<bio_err> and B<prog> parameters (unless
 NULL) to print additional diagnostic information in a user-oriented way.
 
-OSSL_HTTP_parse_url() parses its input string B<url> as a URL and splits it up
+OSSL_HTTP_parse_url() parses its input string B<url> as a URL
+of the form C<[http[s]://]address[:port][/path]> and splits it up
 into host, port and path components and a flag whether it begins with 'https'.
-The host component may be a DNS name or an IPv4 or an IPv6 address.
+The host component may be a DNS name or an IPv4 address
+or an IPv6 address enclosed in C<[> and C<]>.
```

Comment:
Are all 3 enclosed in [] or just IPv6?  Not sure how to read this.

Reply:
I've re-phrased this to:
```
The host component may be a DNS name or an IP address
where IPv6 addresses should be enclosed in square brackets C<[> and C<]>.
```